config/sites/wellbeing.uiowa.edu/config_split.patch.core.entity_form_display.node.page.default.yml

```diff
@@ -4,6 +4,7 @@ adding:
       - field.field.node.page.field_wellbeing_audience
       - field.field.node.page.field_wellbeing_topic
       - field.field.node.page.field_wellbeing_type
+      - workflows.workflow.editorial
```

Comment:
Same with this one. Odd.

Reply:
See my comment https://github.com/uiowa/uiowa/pull/7545#discussion_r1571338956

config/default/core.entity_form_display.node.page.default.yml

Reply:
Resolved. See comment: https://github.com/uiowa/uiowa/pull/7545#discussion_r1571371995